deepchecks/vision/checks/data_integrity/abstract_property_outliers.py

```diff
@@ -50,6 +52,8 @@ class AbstractPropertyOutliers(SingleDatasetCheck):
         - 'categorical' - for discrete, non-ordinal outputs. These can still be numbers,
           but these numbers do not have inherent value.
         For more on image / label properties, see the :ref:`property guide </user-guide/vision/vision_properties.rst>`
+    property_input_type: PropertiesInputType, default: PropertiesInputType.OTHER
```

Comment:
Now that I think about it - in what case will the user want to modify the property_input_type of the check? (of the inheriting check, not this abstract class).
I mean, for example the image property outliers always passes the images to the property, so modifying this argument will just lead to the properties not to be listed under then right section...

Reply:
@noamzbr I don't have an example use case but IMO it's neater that this class is in the same format of the child classes